### PR TITLE
Support systemd and fix the build on Ubuntu 16.04

### LINKED ISSUE #60: Ubuntu 16.04 LTS fails to install.
```
----------------------------------------------------------------------
Preparation complete. Building and packaging xrdp...
----------------------------------------------------------------------
dpkg-buildpackage: warning: using a gain-root-command while being root
tail: cannot open 'debian/changelog' for reading: No such file or directory
dpkg-buildpackage: error: tail of debian/changelog gave error exit status 1
mv: cannot stat 'xrdp*.deb': No such file or directory
./X11rdp-o-matic.sh: line 993: /etc/init.d/xrdp: No such file or directory
dpkg: error processing archive /home/ovan/X11RDP-o-Matic/packages/Xorg/*.deb (--install):
```

### PR DESCRIPTION
**Pull request by metalefty**
Call for volunteers.  Would someone like to test this branch?

```
$ git clone -b ubuntu-16.04 https://github.com/metalefty/X11RDP-o-Matic.git
$ cd X11RDP-o-Matic
# ./X11RDP-o-Matic.sh --justdoit --branch devel  # master branch will fail, not a o-Matic bug.
```

If you have time, results of with `--nox11rdp` are welcome.

```
# ./X11RDP-o-Matic.sh --justdoit --branch devel --nox11rdp
````

At least build should be succeed on: 
* Ubuntu 16.04, 15.10, 15.04, 14.04, 12.04
* Debian GNU/Linux 8.0, 7.0

At least these two points should be checked:
* build successfully finished with no errors
* xrdp is running when o-Matic successfully finished

If no problem, I'll merge this and close #60.

### CODE REVIEW

**Comment by nerdknight** [comment]
On Ubuntu 16.04 failed, here is the output: 
```
config.status: creating config_ac.h
config.status: executing depfiles commands
config.status: executing libtool commands
| dh_make --single --copyright apache --createorig
./X11rdp-o-matic.sh: line 520: cd: debian: No such file or directory
```

File: X11rdp-o-matic.sh [hunk 1]
```diff
@@ -130,6 +130,11 @@ else
   USING_SYSTEMD=true
 fi
 
+# change dh_make option depending on if dh_make supports -y option
+dh_make -h | grep -q -- -y && \
```
Comment (metalefty) [inline X11rdp-o-matic.sh]: dh_make should be installed before doing this.

**Comment by metalefty** [comment]
Run X11RDP-o-Matic.sh again, it will succeed.

But this is indeed a bug. I'll fix it later.

**Comment by metalefty** [comment]
@nerdknight  Fixed. Deinstall dh-make package and could you try again?

**Comment by coolemza** [comment]
xubuntu 16.04 - everything ok

**Comment by nerdknight** [comment]
It works ok now on Ubuntu 16.04

**Comment by metalefty** [comment]
Thanks, still waiting for testing on older Ubuntu and Debian.

**Comment by nerdknight** [comment]
Works ok on Ubuntu 14.04

**Comment by metalefty** [comment]
Works on Debian 8.4.

**Comment by metalefty** [comment]
I'll merge develop into master in a few days.

**Comment by johnpeak** [comment]
This looks like it was merged back into the devel branch, but I still get same error on debian 8.7.  I clone the repo and ran "./X11RDP-o-matic.sh --justdoit --branch devel".  I suspect I'm doing something wrong though.  Any ideas?

**Comment by ZevSuk** [comment]
run "git clone -b ubuntu-16.04 https://github.com/metalefty/XllRDP-o-Matic.git"
Cloning into "XllRDP-o-Matic'...
fatal: Remote branch ubuntu-16.04 not found in upstream origin

I still did not understand.
Is there a support for ubuntu-16 x64 or not?

**Pull request merged**